source/designers-guide/smarty-plugins/index.md

```diff
@@ -0,0 +1,213 @@
+title: Smarty Plugins
+github_link: designers-guide/smarty-plugins/index.md
+indexed: true
+---
+
+<div class="toc-list"></div>
+
+# Smarty plugins #
+
+Several more Smarty plugins are available in the current Shopware version. This makes it even easier to deal with currencies, data (times), configurations, links and paths in the template. Below you will find all the information needed to use these plugins in your Smarty template.
+
+## Currency plugin ##
+
+The currency plugin allows you to control how prices are formatted and displayed in your shop. So you have the possibility to apply different predefined standard formats and to determine where, for example, the currency symbol will appear.
+
+To activate the plugin, please use the following syntax:
+
+Example: *currency syntax*
+```
+{* Syntax *}
+{[Price]|currency:[FORMAT]:[CURRENCY POSITION}
+ 
+{* Examplecall - Output: 49,95 EUR *}
+{$sArticle.price|currency:use_shortname:right}
+```
+
+The following formats are available in the menu:
+
+- no_symbol - price without specifying currency, e.g., 49.95. Here the position is ignored.
+- use_symbol - price with currency symbol, e.g., $ 49.95
+- use_shortname - price with indication of currency as short, e.g., 49.95 USD
+- use_name - price with indication of currency as long as the form, e.g., 49.95 dollars
+
+The following positions are available in the menu:
+
+- left - displays the currency symbol to the left of the price as USD 49.95
+- right - displays the currency symbol to the right of the price as 49.95 USD
+- standard - displays the currency symbol in the default location, which is on the right e.g., 49,95 USD
+
+<div class="alert alert-info">
+	Note: Shopware, by default, formats prices in the frontend as follows:
+	{$sArticle.price|currency:use_symbol:right}.
+	If you wish to have certain areas of your shop in a different format, this plugin will allow you to change the format.
+</div>
+
+## Config plugin ##
+
+The config plugin is grants you the ability to access and read the various features in the backend.
+
+To do so, use the following syntax:
+ 
+Example: *config syntax*
+```
+{* Syntax *}
+{config name=[NAME OF PROPERTY]}
+ 
+{* Examplecall "Disable vote" *}
+{config name=VoteDisable}
+```
+ 
+## Link Plugin ##
+
+The link plugin helps you to specify file paths, for example, when loading images or style sheets. We'll assume that you've installed your shop in a subdirectory shopware on your server (domain = http://www.meinshop.de) and would like to load a style sheet my_styles.css from the directory frontend/_resources/styles into your template my_template.
+
+Without the plugin, the call looks like this:
+
+Example: *normal style sheet call*
+```
+<link rel="stylesheet" media="screen, projection" href="http://www.meinshop.de/shopware/templates/my_template/frontend/_resources/styles/my_styles.css" />
+```
+
+
+By using the link plugin, you no longer need to specify an exact path, merely to go through the selected template.
```

Comment:
"By using the link plugin you only have to define the path relative to your own theme directory."